tasks/move-granules/index.js

```diff
@@ -136,6 +137,11 @@ async function updateGranuleMetadata(granulesObject, collection, cmrFiles, bucke
     const cmrMetadata = cmrFile ? await metadataObjectFromCMRFile(cmrFile.filename) : {};
 
     granulesObject[granuleId].files.forEach((file) => {
+      const cmrFileTypeObject = {};
```

Comment:
Currently move granules *adds* CMR metadata when a cmr file type is detected, we should also add file type to the new granule-file here.   

I'm unconvinced this shouldn't be added to the payload when the CMR file is generated, but that's irrelevant to *this* PR.       MoveGranules will respect a fileType already present (e.g. if a cmr file is in the input for some reason, for some workflows).

Reply:
I believe that the first section is something that @mennovandiermen is addressing this sprint.

Reply:
@ifestus  adding fileType or the metadata, or? 

Which issue?

Reply:
nevermind - Looking at the sprint board I misunderstood the scope of his ticket. I thought there was going to be work done removing some of the metadata-adding from `move-granule`, but I was incorrect.

Reply:
Either way, I agree that this is outside the scope of your work in this PR.